### PR TITLE
Change formatting of constitution.

### PR DESCRIPTION
**Pull request by jdcaperon**
This is a formatting update to the constitution. The changes present in this PR do not change the contents of the constitution and only changes the way the constitution is presented.

This is part of an effort to granularise changes made in #8

### CODE REVIEW

File: constitution.md [hunk 1]
```diff
@@ -1,299 +1,262 @@
...
-        3. conducts himself or herself in a manner considered to be injurious or prejudicial to the character or interests of the Club/Society, the management committee shall consider whether the member's membership shall be terminated.
+**Constitution - Affiliated Bodies**
+**Version 1.4**
+**30th of May 2019**
 
-    4. The member concerned shall be given a full and fair opportunity of presenting the member's case and if the management committee resolves to terminate the membership it shall instruct the secretary to advise the member in writing accordingly.
+### 1 Preliminaries
 
-1. Register of Members
+1.1 The Society shall be called “University of Queensland Computing Society”, hereafter referred to as “the Society”
```
Comment (bradleysigma) [inline constitution.md]:
```suggestion
1.1 The Society shall be called “University of Queensland Computing Society”  
```

Comment (jsutton101) [inline constitution.md]: Club/Society was changed to Society here, but not later in the document

File: constitution.md [hunk 2]
```diff
@@ -1,299 +1,262 @@
...
 
-        2. the nomination, which shall be in writing and signed by the member and the member's proposer and seconder, shall be lodged with the secretary before the annual general meeting at which the election is to take place.
-        
-        3. nominations from the floor of the annual general meeting will be accepted
+* any student of the University of Queensland
+* any University of Queensland Academic Staff Member
+* any other members of the University or wider community interested in Club/Society's activities
 
-1. Resignation or Removal from office of Member of Management Committee
+4.2 The Society shall have no less than thirty (30) members.
```
Comment (bradleysigma) [inline constitution.md]:
```suggestion
4.2 The Society shall have no less than:
* Thirty (30) financial members in the case of a subsidised club
```

File: constitution.md [hunk 3]
```diff
@@ -1,299 +1,262 @@
...
+4.3 Fifteen (15) financial members in the case of a publications club
```
Comment (bradleysigma) [inline constitution.md]:
```suggestion
* Fifteen (15) financial members in the case of a publications club
```

File: constitution.md [hunk 4]
```diff
@@ -1,299 +1,262 @@
...
+4.4 Ten (10) financial members in the case of an Ipswich based club.
```
Comment (bradleysigma) [inline constitution.md]:
```suggestion
* Ten (10) financial members in the case of an Ipswich based club.
```

File: constitution.md [hunk 5]
```diff
@@ -1,299 +1,262 @@
...
+4.5 Seventy percent (70%) of the Society's financial members must be University of Queensland Union members.
```
Comment (bradleysigma) [inline constitution.md]:
```suggestion
4.2 Seventy percent (70%) of the Society's financial members must be University of Queensland Union members.
```

File: constitution.md (hunk 1 above)
Comment (jdcaperon) [reply constitution.md]: Is this material enough to not be considered formatting?

Comment (jdcaperon) [reply constitution.md]: Depending on the resolution of the above comment I'll resolve this.

File: constitution.md (hunk 2 above)
Comment (jdcaperon) [reply constitution.md]: Good call.

File: constitution.md (hunk 1 above)
Comment (bradleysigma) [reply constitution.md]: Yes.

Comment (jdcaperon) [reply constitution.md]: Aight, I've removed it from this formatting PR.

Comment (jdcaperon) [reply constitution.md]: See above.

File: constitution.md [hunk 6]
```diff
@@ -1,299 +1,267 @@
...
 
-        2. the nomination, which shall be in writing and signed by the member and the member's proposer and seconder, shall be lodged with the secretary before the annual general meeting at which the election is to take place.
-        
-        3. nominations from the floor of the annual general meeting will be accepted
+* Thirty (30) financial members in the case of a subsidised club
+* Fifteen (15) financial members in the case of a publications club
+* Ten (10) financial members in the case of an Ipswich based club.
 
-1. Resignation or Removal from office of Member of Management Committee
+4.3 Seventy percent (70%) of the Society's financial members must be University of Queensland Union members.
```
Comment (bradleysigma) [inline constitution.md]:
```suggestion
4.3 70% of the Club/Society's membership must be students.
```

**Comment by ElladeLore** [comment]
lgtm

**Comment by joeladdison** [comment]
Unless I am completely missing something, this is not just a formatting change. The version is changing, as is a huge part of the wording. I echo comments from #8 that there should not be any wording changes associated with a formatting change.

For a formatting change, I should be able to see a side by side diff with no wording changes, just formatting indicators changing. This is not the case for this PR.

If you want to change the contents, then submit them as a completely separate PR. I think there should also be proof attached to the PR that the amendments have been voted on and passed at either a Special General Meeting or Annual General Meeting before such changes are merged, as per the requirements for making such amendments.

**Comment by jdcaperon** [comment]
Completely agree. Unfortunately, I do not know enough about the diffing process to coerce it to behave nicely with the formatting change. Hence, we have the current situation. Anyone who is interested in assisting to make the diff "nicer" or to prove the change is welcome to. It is my current opinion that in the future any changes will present a "nicer" diff. 

As this PR is not a rule change and is simply a formatting change, it is my opinion that the current constitution allows for the formatting change to be made by direction of the committee. As long as there are no changes to the rules. This is afforded to the committee in the current rule `8.i.b`.

**Comment by jdcaperon** [comment]
image
Here is an example diff after the formatting changes are made.

**Comment by jdcaperon** [comment]
I have made a more "diffable" version of the changes.

**Pull request merged**